helm/cluster/templates/clusterapi/_helpers_prekubeadmcommands.tpl

```diff
@@ -0,0 +1,34 @@
+{{- define "cluster.internal.kubeadm.preKubeadmCommands" }}
```

Comment:
Helper templates for specifying pre-kubeadm commands (commands that run before kubeadm in all nodes) which are common for both control plane (KubeadmControlPlane) and worker nodes (KubeadmConfig).